akka-actor/src/main/resources/reference.conf

```diff
@@ -557,6 +557,9 @@ akka {
       # The maximum number of bytes delivered by a `Received` message. Before
       # more data is read from the network the connection actor will try to
       # do other work.
+      # The purpose of this setting is to impose a smaller limit than the 
+      # configured receive buffer size. When using value 'unlimited' it will
+      # try to read all from the receive buffer.
       max-received-message-size = unlimited
```

Comment:
yes, I’d think so: while this is `unlimited` the true limit is controlled in exactly one place—preferable to having two places to modify